[DOC] Updated mirror maker docs (new options added) (#1768)
strimzi/strimzi-kafka-operator: documentation/book/assembly-kafka-mirror-maker-abort-on-send-failure.adoc

// This assembly is included in the following assemblies:
//
// assembly-deployment-configuration-kafka-mirror-maker.adoc

// Save the context of the assembly that is including this one.
// This is necessary for including assemblies in assemblies.
// See also the complementary step on the last line of this file.

[id='assembly-kafka-mirror-maker-abort-on-send-failure-{context}']

> = Abort Mirror Maker on send failure

Reviewer: ```suggestion
= Abort Mirror Maker if message sending fails
```

> You can set whether should be Mirror Maker container terminated after error while sending a message.

Reviewer 2: ```suggestion
You can set whether the Mirror Maker container is terminated following an error while sending a message.
```

> You can set whether the Mirror Maker container is terminated following an error while sending a message.

Reviewer: ```suggestion
You can control whether the Mirror Maker container is terminated if an error occurs when sending a message.
```

> Such a failure at the {ProductPlatformName} causes the container is recreated.

Reviewer 2: ```suggestion
The container is recreated for such a failure on {ProductPlatformName}.
```

> The container is recreated for such a failure on {ProductPlatformName}.

Reviewer: ```suggestion
After the Mirror Maker container has been terminated, it is recreated on {ProductPlatformName}.
```

> The sending error is ignored when this option is set to `false`.

Reviewer 2: ```suggestion
The error sent is ignored when this option is set to `false`.
```

> The error sent is ignored when this option is set to `false`.

Reviewer 3: I think it's not clear what "this option" is referring. There is no reference to the option name which is abortonsendfailure.

Reviewer 3: If it's referring to the title, I don't know .. it doesn't sound clear to me.

Author: @PaulRMellor can you help us to find better formulation?

Reviewer: Good point, Paolo. I suggest rephrasing as follows:

If the `abortonsendfailure` option is set to `false`, message sending errors are ignored.

Reviewer 3: Yeah it sounds better.

Author: Fixed.
